packages/core/src/indexing/postgres/migrations/1-create-model-table.ts

```diff
@@ -72,3 +72,30 @@ export async function createModelTable(
     )
   })
 }
+
+export async function createConfigTable(dataSource: Knex, tableName: string) {
+  switch (tableName) {
+    case 'ceramic_models':
+      await dataSource.schema.createTable(tableName, function (table) {
+        // create model indexing configuration table
+        table.bigIncrements('index_id')
+        table.string('model', 1024).notNullable()
+        table.boolean('is_indexed').notNullable().defaultTo(true)
+        table.dateTime('created_at').notNullable().defaultTo(dataSource.fn.now())
+        table.dateTime('updated_at').notNullable().defaultTo(dataSource.fn.now())
+        table.string('updated_by', 1024).notNullable()
+
+        table.index(['model'], `idx_ceramic_models_model`, {
```

Comment:
why do we need this index if we have the compound index below?  I think this is just redundant storage for no value

Reply:
Or actually I think we may just want the second index to be just on `is_indexed`.  Otherwise we can't efficiently do a query for all indexed models.  I can't think of a time where you'd want to query on `model` AND `is_indexed` together anyway